feature-utils/poly-look/test/utils/history-stub.js

```diff
@@ -0,0 +1,19 @@
+export class HistoryStub {
+  constructor() {
+    this._route = null;
+    this._state = null;
+  }
+
+  get route() {
+    return this._route;
+  }
+
+  get state() {
+    return this._state;
+  }
+
+  push(route, state) {
+    this._route = route;
+    this._state = state;
```

Comment:
Maybe check that state is correct, or at least has the attribute we need?
Also, is this really pushing? It's just setting the state...